### ISSUE TITLE
Getting Voice Notification Eventhough call is active

### ISSUE THREAD

**Issue by wmt-raj-mobile** [cybex-dev/twilio_voice#206]
How do i restrict call notification if call is already active

**Comment by cybex-dev** [comment]
Hi @wmt-raj-mobile 

Thanks for reaching out.

Could you explain / provide more details for "restrict call notifications", what do you have in mind?

**Comment by wmt-raj-mobile** [comment]
If there is ongoing call between person A and person B now if person C calls person A then person A should not receive call notification if there is ongoing call

**Comment by cybex-dev** [comment]
> If there is ongoing call between person A and person B now if person C calls person A then person A should not receive call notification if there is ongoing call

If that could be done, how do you expect to handle subsequent calls or would you simply choose to ignore them?

**Comment by wmt-raj-mobile** [comment]
Yes i want to ignore that. Can you please help me with that? Also Please give exact location where i have to change the code

**Comment by cybex-dev** [comment]
> Yes i want to ignore that. Can you please help me with that

I see - could you provide a sample use-case for this as it is extremely niche.

**Comment by wmt-raj-mobile** [comment]
so our user can have multiple login with same twilio number. if 2 user is using the same number and there is ongoing call with one of that user then that user should not receive call notification. Basically i don't want calll notification if there is ongoing call.

**Comment by wmt-raj-mobile** [comment]
Any update please help

**Comment by cybex-dev** [comment]
Hi @wmt-raj-mobile,

This is clearly a niche feature - moreover it can be handled on the backend/TwiML app using unique client Ids to direct calls. I'll look into adding an "ignoreIncomingCalls` feature but do not expect this any time soon.

You are more than welcome to submit a PR for this in the mean time.

**Comment by wmt-raj-mobile** [comment]
In IOS native code i added "" if(call != nil) {return} "" inside callInviteReceived method before reportIncomingCall method and it satisfied my requirement but don't know about android

**Comment by cybex-dev** [comment]
@wmt-raj-mobile thanks - though I'd need to explore in more detail the consequences of adding this, since some might not require this feature to begin with (as is my use-case).

**Comment by wmt-raj-mobile** [comment]
In android its merging call by itself, if call is already active and someone calls on same number

**Comment by cybex-dev** [comment]
Hi @wmt-raj-mobile,

 I've added a feature to allow multiple calls while already on a call. See #329. If the call is merging, this may be another problem. Take a look at >=0.4.0 to see if the problem persists.

**Comment by cybex-dev** [comment]
Also Available in >=0.5.0, closing for now.
